File: backendworker/NHSOnline.Backend.Worker/Areas/Session/Models/UserSessionResponse.cs
using System;

namespace NHSOnline.Backend.Worker.Areas.Session.Models
{
    public class UserSessionResponse
    {
        public string Name { get; set; }
        
        public int SessionTimeout { get; set; }

        public string OdsCode { get; set; }

        public string Token { get; set; }
        
        public DateTime DateOfBirth { get; set; }
        
        public string NhsNumber { get; set; }
    }
}

File: bddtests/src/main/kotlin/worker/models/session/UserSessionResponse.kt
package worker.models.session

import worker.models.serviceJourneyRules.ServiceJourneyRulesResponse
import javax.servlet.http.Cookie

data class UserSessionResponse(var userSessionResponseCookie: UserSessionResponseCookie,
                               var userSessionResponseBody: UserSessionResponseBody) {

    class UserSessionResponseCookie(var cookie: Cookie)

    class UserSessionResponseBody(var name: String,
                                  var sessionTimeout: Number,
                                  var serviceJourneyRules: ServiceJourneyRulesResponse,
                                  var userSessionCreateReferenceCode: String?)
}
